include/uv/win.h

```diff
@@ -684,6 +684,7 @@ typedef struct {
 #define UV_FS_O_TEMPORARY    _O_TEMPORARY
 #define UV_FS_O_TRUNC        _O_TRUNC
 #define UV_FS_O_WRONLY       _O_WRONLY
+#define UV_FS_O_SHARE_RDONLY 0x40000000 /* READ ONLY SHARING MODE */
```

Comment:
Define this in include/uv/unix.h as well but with a value of 0.

The comment should probably say just `/* FILE_SHARE_READ */`?